### ISSUE TITLE
Change to VASSAL3.5.0

### ISSUE THREAD

**Issue by derimmer** [vasl-developers/vasl#833]

**Comment by derimmer** [comment]
@uckelman I am trying to move to 3.5.3. I do this by first updating the pom to use 3.5.3 and then by changing the dependency in the Project Structure from vassal-app-3.5.0.jar to vassal-app-3.5.3.jar. I can remove vassal-app-3.5.0.jar as per normal but when it comes to adding the vassal-app-3.5.3.jar, I cannot find it. In previous updates, I have navigated to C:\Users\<myname>\.m2\repository\org\vassalengine\vassal-app and then selected the version folder and then the jar. However, my vassal-app folder is not showing any versions after 3.5.0. I have installed 3.5.3 on my machine. 

I suspect I am missing a simple step here but I cannot recall what it is.

**Comment by uckelman** [comment]
Updating the pom.xml to 3.5.3 is the right thing. 

> and then by changing the dependency in the Project Structure from vassal-app-3.5.0.jar to vassal-app-3.5.3.jar. I can remove vassal-app-3.5.0.jar as per normal but when it comes to adding the vassal-app-3.5.3.jar, I cannot find it. In previous updates, I have navigated to C:\Users<myname>.m2\repository\org\vassalengine\vassal-app and then selected the version folder and then the jar. However, my vassal-app folder is not showing any versions after 3.5.0. I have installed 3.5.3 on my machine.

I don't know what you're referring to here. Maven should fetch the JARs you need based on the pom.xml.

**Comment by derimmer** [comment]
Apologies, I was not clearly differentiating between building the VASL module with VASSAL3.5.3 and updating my development environment. It was the later that was causing me a problem. However, when I went ahead and built VASL6.6.2 using Maven, it updated the vassal-app directory that I was referring to. So now I have built the module and got my dev environment using 3.5.3. 

I await 3.5.4 and following! LOL. Have written note to self to remember how to do this properly.
